releasenotes/notes/add-sparse-torch-connector-a3b9e3d50b405a01.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    The PyTorch connector :class:`~qiskit_machine_learning.connector.TorchConnector` supports
+    sparse output in both passes. That means a hybrid network that has a quantum layer must rely
+    on PyTorch sparse operations only.
```

Comment:
Do you think this enough for the release note for this change? Would an example improve it? 

What does this mean for an end user seeing this.

>That means a hybrid network that has a quantum layer must rely on PyTorch sparse operations only.

If I have a regular PyTorch network, without quantum, is this still a true statement. I am trying to figure as an end user who is going to create a hybrid network what this means to me say as different from a pure classical set that I might have used before.

Reply:
Updated the reno: added an example and rephrased this limitation. Hopefully, it is better now.